### PR TITLE
Fix staging broken

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-152
It seems that some images without orientation will throw error in `image.getexif()`, but no exception log :sadfrog:

### CODE REVIEW

File: mockup_package/mockup/image_generator.py [hunk 1]
```diff
@@ -43,17 +43,20 @@ def create_fit_resolution_image(self):
...
+
+            exif = image.getexif()
+            if exif[orientation] == 3:
+                image = image.rotate(180, expand=True)
+            elif exif[orientation] == 6:
+                image = image.rotate(270, expand=True)
+            elif exif[orientation] == 8:
+                image = image.rotate(90, expand=True)
+        except:
+            pass
```
Comment (pkong-ds) [inline mockup_package/mockup/image_generator.py]:
```suggestion
          except SpecificOrientationException:
            # image does not have orientation, expected case, will silent error
            pass
```

**Review by pkong-ds — CHANGES_REQUESTED** [review]

File: mockup_package/mockup/image_generator.py (hunk 1 above)
Comment (YayunHuang) [reply mockup_package/mockup/image_generator.py]: @pkong-ds added comment, but I'm just using `Exception` as not sure what exception it sends

Comment (pkong-ds) [reply mockup_package/mockup/image_generator.py]: @YayunHuang I added a commit for this. Can help ~review~ merge if ok?

**Review by pkong-ds — APPROVED** [review]

File: mockup_package/mockup/image_generator.py (hunk 1 above)
Comment (pkong-ds) [reply mockup_package/mockup/image_generator.py]: btw the trick I used to print the error https://stackoverflow.com/a/9824050/19287186

**Pull request merged**